### DISCUSSION TITLE
Why can't apps have the "manage staff" permissions?

### DISCUSSION THREAD

**Discussion by rrustvold** [saleor/saleor#7936] [Q&A]
Line 329 of saleor/graphql/core/mutations.py notes that "for now MANAGE_STAFF permission for app is not supported" and will return permission denied for any app token that is attempting to utilize that permission. This is true even if the app has been given that permission in the dashboard. I'm wondering why this is? Can it be removed?

**Comment by korycins** [comment]
Hey @rrustvold ,
Thanks for raising this. I think this can be removed. Previously, App with manage_staff could create users with bigger permission than app's permissions, but this is already fixed.
